hack/run-e2e-conformance-virtual-cluster.sh

```diff
@@ -130,18 +130,67 @@ kcli ssh $cluster_name-ctlplane-0 << EOF
 sudo su
 echo '$insecure_registry' > /etc/containers/registries.conf.d/003-internal.conf
 systemctl restart crio
+
+echo '[connection]
```

Comment:
nit: define function and call it for each node ?

Reply:
done